src/Command/ErrorFormatter/SarifErrorFormatter.php

```diff
@@ -0,0 +1,124 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Command\ErrorFormatter;
+
+use PHPStan\Command\AnalysisResult;
+use PHPStan\Command\Output;
+use PHPStan\File\RelativePathHelper;
+use PHPStan\Internal\ComposerHelper;
+use function json_encode;
+use const JSON_HEX_AMP;
+use const JSON_HEX_APOS;
+use const JSON_HEX_QUOT;
+use const JSON_HEX_TAG;
+use const JSON_PRETTY_PRINT;
+
+class SarifErrorFormatter implements ErrorFormatter
+{
+
+	private const URI_BASE_ID = 'WORKINGDIR';
+
+	public function __construct(
+		private RelativePathHelper $relativePathHelper,
+		private string $currentWorkingDirectory,
+		private bool $pretty,
+	)
+	{
+	}
+
+	public function formatErrors(AnalysisResult $analysisResult, Output $output): int
+	{
+		$phpstanVersion = ComposerHelper::getPhpStanVersion();
+
+		$tool = [
+			'driver' => [
+				'name' => 'PHPStan',
+				'fullName' => 'PHP Static Analysis Tool',
```

Comment:
This isn't the name of the tool. Just use `PHPStan` too.

Reply:
Fixed in 4503bb7d594c075c3dd0eceef3205a795c8e288a